config/initializers/figaro.rb

```diff
@@ -6,7 +6,6 @@
   'domain_name',
   'enable_identity_verification',
   'enable_test_routes',
-  'equifax_ssh_passphrase',
```

Comment:
I had hoped the commit message "from RC" was clear enough. I have commented out instead of deleting as well.